homeassistant/components/devolo_home_control/config_flow.py

```diff
@@ -29,22 +35,35 @@ def __init__(self):
     async def async_step_user(self, user_input=None):
         """Handle a flow initiated by the user."""
         if self.show_advanced_options:
-            self.data_schema = {
-                vol.Required(CONF_USERNAME): str,
-                vol.Required(CONF_PASSWORD): str,
-                vol.Required(CONF_MYDEVOLO, default=DEFAULT_MYDEVOLO): str,
-            }
+            self.data_schema.update(
+                {
+                    vol.Required(CONF_MYDEVOLO, default=DEFAULT_MYDEVOLO): str,
+                }
+            )
```

Comment:
```suggestion
            self.data_schema[vol.Required(CONF_MYDEVOLO, default=DEFAULT_MYDEVOLO)] = str
```

Not 100% sure this will work, but much shorter if it does

Reply:
It does and indeed looks better.